src/features/views/l10n/messageIds.ts

```diff
@@ -30,6 +30,8 @@ export default makeMessages('feat.views', {
     },
     menu: {
       delete: m('Delete'),
+      duplicate: m('Duplicate'),
+      folderCopy: m<{ folderName: string }>('{folderName} - copy'),
```

Comment:
I think this way of naming the copied view is good, i.e. the language of adding `- copy`. But the name of the message, `folderCopy` and the name of the parameter `folderName`, feel odd to me, because this is not related to folders.

Reply:
Updated it now :)